spec/controllers/organizations_controller_spec.rb

```diff
@@ -34,6 +35,22 @@
       end
     end
 
+    context 'user not in the classroom' do
+      before do
+        sign_in(student)
+      end
+
+      it 'will be added if he is admin of the GitHub organization' do
+        get :index
+        expect(assigns(:current_user).organizations.first.id).to eq(organization.id)
+      end
+
+      it 'will not be added if he is not admin of the GitHub organization' do
+        get :index
+        expect(assigns(:current_user).organizations.length).to eq(0)
+      end
+    end
+
```

Comment:
"apart" should be "part"

```ruby
      it 'will add the user to the classroom' do
        expect(user.organizations).to be_empty

        get :index

        expect(user.organizations).not_to be_empty
        expect(user.organizations).to include(organization)
      end
```

one expectation is enough here: `expect(user.organizations).to include(organization)`

```ruby
      before(:each) do
        sign_out
        sign_in(student)
      end
```

I really don't like that we're signing out and then back in here. Can we modify the nesting so the sign out isn't necessary?

"will not add the user to the classroom" should be "does not" and only one expectation is needed here.